src/CTA.WebForms2Blazor/ProjectManagement/ProjectBuilder.cs

```diff
@@ -20,10 +21,17 @@ public ProjectBuilder(string outputProjectPath)
 
         public void CreateRelativeDirectoryIfNotExists(string relativePath)
         {
-            var fullPath = Path.Combine(_outputProjectPath, relativePath);
+            try
+            {
+                var fullPath = Path.Combine(_outputProjectPath, relativePath);
 
-            // No exception is thrown if the directory already exists
-            Directory.CreateDirectory(fullPath);
+                // No exception is thrown if the directory already exists
+                Directory.CreateDirectory(fullPath);
+            }
+            catch (Exception e)
+            {
+                LogHelper.LogError(e, "Could not create relative directory.");
```

Comment:
May want to record the directory location in the error message here

Reply:
The path might be recorded by the exception itself, but will add here explicitly just in case